pkg/rules/terraswap/utils.go

```diff
@@ -8,3 +8,12 @@ func IsFactoryAddress(address string) bool {
 	}
 	return false
 }
+
+func TerraswapTypeOf(address string) TerraswapType {
+	for t, v := range FactoryAddress {
```

Comment:
`FactoryAddress` doesn't seem to be used anywhere else. whatabout replacing key to value?

Reply:
I move the location with small refactoring https://github.com/dezswap/cosmwasm-etl/pull/23/commits/81dc1b95c156bbfc0a9c83deca9df71f1f8ab64f
(It is time to refactor the related package... thank you)